### DISCUSSION TITLE
Set preset/set of rules in global configuration for specific repositories only

### DISCUSSION THREAD

**Discussion by XKrylus** [renovatebot/renovate#22888] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

Renovate 35:133.3

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I want to specify a subset of repositories that are being managed by renovatebot which group all updates to single merge request. I would prefer to instead of setting the 'group:all' preset individually in each of those repos (15+) to have this configuration defined in renovatebot renovaterc.json and define which repositories should use this preset/configuration there.

### What you tried so far.

I have searched for a way to define preset globally for specific projects, which I am not sure is possible. I have also attempted to set the 'packageRules' with 'matchRepos' rule to define grouping for specific repos, but this seems to then ignore the rest of repositories, where I want to keep the policy of 1 merge request per update

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment]
repositories global option is an array of objects, you can define it there with repo name. check the docs.

**Reply by rarkins** [reply]
That won't work on the repository level though - it would need to be a global preset
